hs_core/models.py

```diff
@@ -1024,6 +1025,56 @@ def update(cls, element_id, **kwargs):
     def remove(cls, element_id):
         raise ValidationError("Coverage element can't be deleted.")
 
+    def add_to_xml_container(self, container):
```

Comment:
I have not been able come up with a better serialization pattern than this. Basically, I am making each metadata element class to serialize itself using the given xml node.  Which I think is much better than what we have now.